lib/matplotlib/rcsetup.py

```diff
@@ -409,7 +407,15 @@ def deprecate_axes_colorcycle(value):
 validate_colorlist = _listify_validator(validate_color, allow_stringlist=True)
 validate_colorlist.__doc__ = 'return a list of colorspecs'
 
-validate_stringlist = _listify_validator(six.text_type)
+def validate_string(s):
+    if isinstance(s, str):  # Always leave str as str
```

Comment:
isinstance(s, (str, six.text_type))

Reply:
Yeah, I left the separate cases here since I thought it made the intent clearer, but it could just as easily be combined.